src/components/Explore/Explore.tsx

```diff
@@ -68,22 +73,37 @@ const Explore: React.FunctionComponent<{
       <Styles.ChartControlsContainer>
         <Legend series={series} />
       </Styles.ChartControlsContainer>
-      <Styles.ChartContainer>
-        {/* The width is set to zero while the parent div is rendering */}
-        <ParentSize>
-          {({ width }) =>
-            width < 10 ? null : (
+      {hasData ? (
+        <Styles.ChartContainer>
+          {/* The width is set to zero while the parent div is rendering */}
+          <ParentSize>
+            {({ width }) => (
               <ExploreChart
                 series={series}
                 isMobile={isMobile}
                 width={width}
                 height={400}
                 tooltipSubtext={`in ${locationName}`}
               />
-            )
-          }
-        </ParentSize>
-      </Styles.ChartContainer>
+            )}
+          </ParentSize>
+        </Styles.ChartContainer>
+      ) : (
+        <EmptyChart height={400}>
```

Comment:
FYI- I put the `<ScreenshotReady/>` marker inside `ExploreChart` so if somebody tried to share an empty chart, the screenshot will fail...  but probably nobody will do that, so it probably doesn't matter.  But it might be sane to move the marker up to this level.

Reply:
I'm adding it to the empty chart just in case